modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -451,7 +601,17 @@ The following table lists the known issues and the version in which they were fi
 |===
 | Known issues | Fixed in
 
-| Vector indexes cannot be assigned autogenerated names. 
+| Procedure signatures from `SHOW PROCEDURES` will render the vector arguments with a type of `ANY` rather than the semantically correct type of `LIST<INTEGER \| FLOAT>`.
+[NOTE]
+====
+The types are still enforced as `LIST<INTEGER \| FLOAT>`.
+====
+|
```

Comment:
if it isn't fixed should it be up in the `Limitations and idiosyncrasies` instead of this table (from what I've seen this holds the fixed issues and that the open ones?)

Reply:
We will keep it here for now, as it still might be fixed - but will change to your suggestion if fix becomes less probable